docs/user_manual/type_data.md

```diff
@@ -0,0 +1,514 @@
+the Create Tissue Material page mentioned above. Keep in mind that any changes you make will affect all samples that
+are already using the affected tissue materials.
+
+MISO administrators can also delete tissue materials. To do so, go to the Tissue Materials list page, check the
+checkboxes beside the materials you would like to delete, and click the "Delete" button in the toolbar at the top of
+the table. You will only be allowed to delete a tissue material if the option has not been used by any existing
+samples.
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Tissue Origins" %}
+
+Note: This item only applies if your site uses [detailed sample mode](site_configuration.html#detailed_sample_mode).
+
+Tissue origins describe what part of the donor or organism the sample was taken from. For human donors, options may
+include 'Brain,' 'Lung,' and 'Pancreas.'
+
+The Tissue Origins list page shows all of the available options. To get to this page, click "Tissue Origins" in the
+Institute Defaults list near the bottom of the menu on the left side of the screen. Tissue origins may only be modified
+by MISO administrators. If you are a MISO administrator, the Tissue Origins list page offers controls for doing so.
+
+To add new tissue origins, go to the Tissue Origins list page and click the "Add" button in the toolbar at the top of
+the table. In the dialog, enter the quantity of new tissue origins that you would like to create and click the "Create"
+button. This will take you to the bulk Create Tissue Origins page. Enter the alias and description you would like for
+each of the new tissue origins and click the "Save" button at the top right to confirm.
+
+To edit existing tissue origins, go to the Tissue Origins list page, check the checkboxes next to the tissue origins
+that you would like to modify, and then click the "Edit" button in the toolbar at the top of the table. This will take
+you to the bulk Edit Tissue Origins page, which is identical to the Create Tissue Origins page. Keep in mind that any
+changes you make will affect all samples that are already using the affected tissue origins.
+
+To delete tissue origins, go to the Tissue Origins list page, check the checkboxes beside the origins you would like to
+delete, and click the "Delete" button in the toolbar at the top of the table. You will only be allowed to delete a
+tissue origin if the option has not been used by any existing samples.
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Tissue Types" %}
+
+Note: This item only applies if your site uses [detailed sample mode](site_configuration.html#detailed_sample_mode).
+
+Tissue Type is a classifier for tissue samples. It may be used to identify the use of the sample. For example, tissue
```

Comment:
`the use of the sample`: `use` feels like not quite the right term to me. Maybe `It may be used to categorize or describe the sample` ? (I'm not fully sold on either of these, and this is more a point of discussion than a request for change -- if it makes sense to everyone else then keep it!)

Reply:
"Categorize" works, though it's just as vague as "type." It could be said that every sample field "describes" the sample, so that's getting even more vague. A lot of these terms overlap and it's difficult to describe specifically what each is meant to represent. I don't think "the use of the sample" is a great description either. I was hoping the examples were helpful enough. Open to other suggestions

Reply:
Changing to "may be used to categorize the sample" for now